server/lib/client.py

```diff
@@ -0,0 +1,76 @@
+import socket
+import json
+import pickle
+
+from threading import Thread
+from lib import request_types as request
+
+
+class Client(Thread):
+
+    def __init__(self, connection, address, requests_dictionary, mas_package, server=None):
+        Thread.__init__(self)
+        self.__connection = connection
+        self.__address = address
+        self.__requests_dictionary = requests_dictionary
+        self.__MAX_PACKAGE = mas_package
+        self.__server = server
+        self.__logged = False
+        self.__nick = None
+        self.__end = False
```

Comment:
Literówka mas_package

Reply:
ja ten kod tylko przepisałem to nie moja robota :D
ale już poprawione